tests/screenshotter/backstop.js

```diff
@@ -0,0 +1,65 @@
+const urls = require( './config/config.url.json' );
+
+const getScenarios = () => {
+	const scenarios = [];
+
+	urls.forEach( ( url ) => {
+		scenarios.push( {
+			label: url,
+			url: url,
+			referenceUrl: url,
+			readyEvent: '',
+			readySelector: '',
+			delay: 2000,
+			onBeforeScript: `${ __dirname }/scripts/on-before-ready.js`,
+			onReadyScript: `${ __dirname }/scripts/on-ready.js`,
+			misMatchThreshold: 0,
+			requireSameDimensions: true,
+		} );
+	} );
+
+	return scenarios;
+};
+
+module.exports = {
+	id: 'elementor_screenshotter',
+	viewports: [
+		{
+			label: 'phone',
+			width: 767,
+			height: 575,
+		},
+		{
+			label: 'tablet',
+			width: 1024,
+			height: 768,
+		},
+		{
+			label: 'desktop',
+			width: 1366,
+			height: 768,
+		},
+	],
+	scenarios: getScenarios(),
+	paths: {
+		bitmaps_reference: `backstop_data/reference`,
+		bitmaps_test: `backstop_data/bitmaps_test`,
+		engine_scripts: `backstop_data/engine_scripts`,
+		html_report: `backstop_data/html_report`,
+		ci_report: `backstop_data/ci_report`,
+	},
+	report: [ 'browser', 'CI' ],
+	ciReport: {
+		format: 'junit',
+	},
+	engine: 'puppeteer',
+	engineOptions: {
+		slowMo: 500,
+		args: [ '--no-sandbox' ],
+	},
+	asyncCaptureLimit: 10,
+	asyncCompareLimit: 10,
+	debug: false,
+	debugWindow: false,
+	fileNameTemplate: '{elementor_screenshotter}_{scenarioLabel}_{selectorIndex}_{selectorLabel}_{viewportIndex}_{viewportLabel}',
```

Comment:
Please re-format and rename the files to this example:
`buttons_phone`
`headings_tablet`
all other data in the name should be removed.

Reply:
Fixed